resources/definitions/felixtec4dual.def.json

```diff
@@ -0,0 +1,59 @@
+{
+    "version": 2,
+    "name": "Felix Tec 4 Dual",
+    "inherits": "fdmprinter",
+    "metadata": {
+        "visible": true,
+        "author": "kerog777@gmail.com",
+        "manufacturer": "Felix",
+        "platform": "FelixTec4_platform.stl",
+        "platform_offset": [-128,0, 102],
+        "machine_extruder_trains":
+        {
+            "0": "felixtec4_dual_extruder_0",
+            "1": "felixtec4_dual_extruder_1"
+        },
+        "file_formats": "text/x-gcode",
+        "has_variants": true,
+        "has_materials": true,
+        "preferred_variant_name": "felixtec4_0.35",
```

Comment:
It's a bit confusing but `preferred_variant_name` is actually the name. So here it should be `0.25 mm`

Reply:
Ah, sorry, it's "0.35 mm"